### ISSUE TITLE
can't login since 08/19/2020 -- might need 1.6

### ISSUE THREAD

**Issue by michael-j-oconnor** [tripplet/skypeweb4adium#14]
I'm unable to sign in since yesterday.  Apparently, that's true of people using the Pidgin plugin from which this is derived.  There's been a report that 1.6 + alternate login worked for the Pidgin version.  See:

https://github.com/EionRobb/skype4pidgin/issues/656

**Comment by tripplet** [comment]
I have updated skype4pidgin, for me it still does not work but that might be related to two factor authentication, can you try the new [release 1.6](https://github.com/tripplet/skypeweb4adium/releases/tag/1.6)

**Comment by michael-j-oconnor** [comment]
I have a Skype-only account not tied to anything else "Microsoft".  It predates Microsoft's acquisition of Skype.  

1.6 without "Alt Login" seems to work for me.  Yay!

1.6 with "Alt Login" doesn't work for me -- get "profile accrual" messages.

It looks like a 1.7 was just released for Pidgin, that might be needed based on account type.

**Comment by tripplet** [comment]
skypeweb4adium v1.6 is based of v1.7 of skype4pidgin (yeah a little confusing).
But since it seems to work and nobody else reported a problem I think this can be closed?

**Comment by EionRobb** [comment]
Thanks for the update @tripplet 👍🏻

**Comment by tripplet** [comment]
No problem, it also worked for me using a generated app password.

**Comment by craftey** [comment]
I have installled (upgraded) to skypeweb4adium 1.6. After restart of Adium (1.5.11b3) I cannot see skype as an option in the account types. my old skypeweb4adium account seems to be removed.

**Comment by tripplet** [comment]
The problem is that I didn't pay the yearly 100$ apple tax so the plugin is not [notarized](https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution) and is blocked by macos by default.
After installing the plugin you have to go into the system settings -> security -> general and then there will be a button to unblock the plugin (https://support.apple.com/en-us/HT202491).
After that restart adium and the plugin should be available again.

**Comment by EionRobb** [comment]
@tripplet did you want to be added to an apple dev team so that you can do the notarisation?

**Comment by craftey** [comment]
@tripplet settings -> security -> general does not show anything. Maybe it is because I am on Mojave? Maybe there is a command line that I could execute to unblock the plugin?

**Comment by tripplet** [comment]
The plugin should always be visible in the xtra menu, see video: https://streamable.com/afj69k
It should work also work on Mojave, but I never tested it with 32bit are you running under 64bit.
@EionRobb That would certainly be helpful but I don't want to be a burden on someone.

**Comment by craftey** [comment]
I can see the plugin v 1.6 in the xtras-manager. However I cannot see it in the new account selector. Also I do not get prompted to allow any security measures and settings -> security -> general there is no notice to allow the plugin. This is on Mojave with Adium 1.5.11b3. Activity-Monitor tells Adium is running 64bit.

**Comment by craftey** [comment]
If I let the debug window open on quit it is still open on launch of Adium. I searched for "skype" caseinsensitive and found exactly one entry:
```09:32:37: Available services are (
    "<ESMeanwhileService: serviceCodeUniqueID = libpurple-Sametime; serviceID = Sametime; serviceClass = Sametime; longDescription = Lotus Sametime>",
    "<ESMSNService: serviceCodeUniqueID = libpurple-MSN; serviceID = MSN; serviceClass = MSN; longDescription = MSN Messenger>",
    "<ESAIMService: serviceCodeUniqueID = libpurple-oscar-AIM; serviceID = AIM; serviceClass = AIM-compatible; longDescription = AOL Instant Messenger>",
    "<ESYahooService: serviceCodeUniqueID = libpurple-Yahoo!; serviceID = Yahoo!; serviceClass = Yahoo!; longDescription = Yahoo! Messenger>",
    "<PurpleFacebookService: serviceCodeUniqueID = facebook-http; serviceID = Facebook; serviceClass = Facebook; longDescription = Facebook>",
    "<ESICQService: serviceCodeUniqueID = libpurple-oscar-ICQ; serviceID = ICQ; serviceClass = AIM-compatible; longDescription = ICQ>",
    "<ESJabberService: serviceCodeUniqueID = libpurple-Jabber; serviceID = Jabber; serviceClass = Jabber; longDescription = XMPP (Jabber)>",
    "<ESSIPEService: serviceCodeUniqueID = libpurple-SIPE; serviceID = SIPE; serviceClass = SIPE; longDescription = Office Communicator>",
    "<ESSimpleService: serviceCodeUniqueID = libpurple-simple; serviceID = SIMPLE; serviceClass = SIMPLE; longDescription = SIP / SIMPLE>",
    "<ESIRCService: serviceCodeUniqueID = libpurple-IRC; serviceID = IRC; serviceClass = IRC; longDescription = IRC (Internet Relay Chat)>",
    "<AIGTalkService: serviceCodeUniqueID = libpurple-jabber-gtalk; serviceID = GTalk; serviceClass = Jabber; longDescription = Google Talk>",
    "<AIMobileMeService: serviceCodeUniqueID = libpurple-oscar-MobileMe; serviceID = MobileMe; serviceClass = AIM-compatible; longDescription = MobileMe>",
    "<ESYahooJapanService: serviceCodeUniqueID = libpurple-Yahoo!-Japan; serviceID = Yahoo! Japan; serviceClass = Yahoo! Japan; longDescription = Yahoo! Japan>",
    "<ESNovellService: serviceCodeUniqueID = libpurple-GroupWise; serviceID = GroupWise; serviceClass = GroupWise; longDescription = Novell GroupWise>",
    "<PurpleMySpaceService: serviceCodeUniqueID = libpurple-MySpace; serviceID = MySpace; serviceClass = MySpace; longDescription = MySpaceIM>",
    "<AILiveJournalService: serviceCodeUniqueID = libpurple-jabber-livejournal; serviceID = LiveJournal; serviceClass = Jabber; longDescription = LiveJournal>",
    "<AITwitterService: serviceCodeUniqueID = twitter; serviceID = Twitter; serviceClass = Twitter; longDescription = Twitter>",
    "<ESDotMacService: serviceCodeUniqueID = libpurple-oscar-Mac; serviceID = Mac; serviceClass = AIM-compatible; longDescription = .Mac>",
    "<PurpleSlackService: serviceCodeUniqueID = libpurple-slackweb; serviceID = Slack; serviceClass = Slack; longDescription = Slack API>",
    "<AIFacebookXMPPService: serviceCodeUniqueID = FBXMPP; serviceID = Facebook; serviceClass = Facebook; longDescription = Facebook>",
    "<AWBonjourService: serviceCodeUniqueID = bonjour-libezv; serviceID = Bonjour; serviceClass = Bonjour; longDescription = Bonjour>",
    "<ESGaduGaduService: serviceCodeUniqueID = libpurple-Gadu-Gadu; serviceID = Gadu-Gadu; serviceClass = Gadu-Gadu; longDescription = Gadu-Gadu>",
    "<TelegramService: serviceCodeUniqueID = prpl-telegram; serviceID = Telegram; serviceClass = Telegram; longDescription = Telegram>"
): could not load account {
    ObjectID = 15;
    Service = Skype;
    Type = "libpurple-skypeweb";
    UID = "myusername";
} on service libpurple-skypeweb (service (null))```

**Comment by tripplet** [comment]
That is interesting the problem seems to be that Adium cannot load some settings for the skype account.
Can you use a different user with the OPTION method and "Edit Users..."
<img width="544" alt="image" src="https://user-images.githubusercontent.com/628197/92520102-5f972080-f21b-11ea-8279-3f5b71763a0e.png">
Alternativ can you try it under a different macOS account?

**Comment by michael-j-oconnor** [comment]
Hmmm...  the skypeweb4adium 1.6 plugin stopped working for me last week-ish (was on vacation).
Alt Login vs. not doesn't seem to matter much, other than the error is verbose (still "profile accrual")
I sign in via https://login.skype.com and it seems to work without incident, extra prompts, etc.

If this ought to be forked off to a separate bug, np.  But, what I filed was reopened so putting here for now.

**Comment by craftey** [comment]
@tripplet I have tested with a new adium account, wie the account selector. And I have tested with a new macOS user. In both cases I cannot see skype as an option in the account types. I am trying this with Adium 1.5.11b3 and not the latest stable 1.5.10.4. Maybe that is the issue.
I also found, that another plugin https://github.com/majn/telegram-purple/releases/ does only work in version 1.3.1 with my Adium. If I choose a newer version of that plugin, I see the same behavior: The type is not shown in the list of account types.
I want to use 1.5.11.b3 because there seems another (unrelated) bug fixed, which is not fixed in 1.5.10.4. But I ll try my tests with 1.5.10.4 and report back.

According to this comment https://github.com/majn/telegram-purple/issues/496#issuecomment-604446726 Adium is still not dead. Maybe we see a new release some time in the future.

**Comment by craftey** [comment]
I have tested with Adium 1.5.10.4. Skypeweb4adium 1.6 now shows up in the account settings. But like @michael-j-oconnor noted it currently does not login.

Also telegram plugin work in latest version 1.4.3. So my issue was related to that beta build of Adium. Thanks for your help @tripplet .

**Comment by craftey** [comment]
Right now login via plugin v1.6 works.

**Comment by tripplet** [comment]
> According to this comment majn/telegram-purple#496 (comment) Adium is still not dead. Maybe we see a new release some time in the future.
I wouldn't count on it, Adium seems to be in this limbo for years now, which I find really sad as it is an awesome tool with a solid foundation.

As it seems to work for everyone, I will close the issue.
Feel free to open a new issue if a new (or the same) problem appears again.
